admin/init.sh

```diff
@@ -22,4 +22,4 @@ gpg --list-secret-keys
 mkdir -p ~/.sbt/0.13/plugins
 cp files/gpg.sbt ~/.sbt/0.13/plugins/
 
-export SBT_CMD=$(which sbt)
+export sbtCmd=$(which sbt)
```

Comment:
I’m still confused about this rename. Isn’t SBT_CMD used in the scripts?

Reply:
Yes, the scripts use `SBT_CMD`. However, the `common` script (executed after `init.sh`) defines `SBT_CMD=${sbtCmd-sbt}`, so it uses `sbtCmd` as a default value, but if `sbtCmd` it sets `SBT_CMD=sbt`. If we set `SBT_CMD` in `init.sh` (and leave `sbtCmd` empty), `common` will overwrite it.